sound/soc/sof/debug-dsp-ops.c

```diff
@@ -311,5 +327,13 @@ int sof_dbg_dsp_ops_test_init(struct snd_sof_dev *sdev)
 	if (ret < 0)
 		return ret;
 
-	return sof_dsp_dsp_ops_create_dfse(sdev, "unload_fw", dsp_ops_debugfs, 0222);
+	ret = sof_dsp_dsp_ops_create_dfse(sdev, "unload_fw", dsp_ops_debugfs, 0222);
+	if (ret < 0)
+		return ret;
+
+	ret = sof_dsp_dsp_ops_create_dfse(sdev, "init_trace", dsp_ops_debugfs, 0222);
+	if (ret < 0)
+		return ret;
+
+	return sof_dsp_dsp_ops_create_dfse(sdev, "free_trace", dsp_ops_debugfs, 0222);
```

Comment:
how do you start the trace after powering on the DSP? Usually we need to send IPC messages. Does the `sof_resume()` will be called to take care of it?

Reply:
Our regular sequence for starting trace after boot is: run_firmware() -> trace_init(). 

So, in the test script, we'd do the same. boot the firmware and then call init_trace. And then do whatever else we want to do say for example load libaries, topologies etc

Reply:
Can you share the test script? It is hard to visualize how this is used.

Reply:
Oooohh, so that's why you haven't been reviewing the test script @ujfalusi ? Mere lack of marketing? The PR is here:
- https://github.com/thesofproject/sof-test/pull/1180